vita3k/lang/include/lang/state.h

```diff
@@ -587,6 +587,8 @@ struct LangState {
             { "boot_apps_full_screen", "Boot apps in full screen" },
             { "audio_backend", "Audio Backend" },
             { "select_audio_backend", "Select your preferred audio backend." },
+            { "audio_volume", "Audio Volume" },
+            { "audio_volume_description", "Applies a global volume multiplier to all audio outputs." },
```

Comment:
this is little wrong, is not multipilier, but pourcent ex:
"Applies a certain percentage of the volume to all audio outputs."

Reply:
I put multiplier as it's internally just multiplying by a float (e.g. 50% -> 0.5) but I agree that it should be reworded to match the user-facing control, will update.

Reply:
yes ^^